core/new-gui/src/app/dashboard/component/feature-container/saved-workflow-section/ngbd-modal-workflow-executions/ngbd-modal-workflow-executions.component.ts

```diff
@@ -82,14 +90,21 @@ export class NgbdModalWorkflowExecutionsComponent implements OnInit {
   /* delete a single execution and display current workflow execution */
 
   onDelete(row: WorkflowExecutionsEntry) {
-    if (this.workflow.wid === undefined) {
-      return;
-    }
-    this.workflowExecutionsService
-      .deleteWorkflowExecutions(this.workflow.wid, row.eId)
+    // Confirmation prompt for deletion
+    const modalRef = this.modalService.open(NgbdModalDeleteWorkflowComponent);
+    modalRef.componentInstance.workflow = cloneDeep(row);
```

Comment:
why do you need this line? I don't think it is needed. The row is an execution not a workflow.

Reply:
Also, this will cause the frontend to complain(check the browser console) it expects workflow as an input and it is not passed!. Will be fixed by a future PR to resolve this.